src/Workspaces/Core/Portable/Shared/Extensions/SyntaxGeneratorExtensions.cs

```diff
@@ -190,11 +190,20 @@ private static SyntaxNode CreateNewArgumentNullException(SyntaxGenerator factory
                 : factory.ReferenceEqualsExpression(identifier, nullExpr);
 
             // generates: if (s == null) throw new ArgumentNullException(nameof(s))
-            return factory.IfStatement(
+            var throwStatement = factory.ThrowStatement(CreateNewArgumentNullException(
+                                    factory, semanticModel.Compilation, parameter));
+            throwStatement = throwStatement.WithLeadingTrivia(factory.Whitespace(" ")).WithoutAnnotations(SyntaxAnnotation.ElasticAnnotation);
+
+            var ifStatement = factory.IfStatement(
                condition,
                 SpecializedCollections.SingletonEnumerable(
-                    factory.ThrowStatement(CreateNewArgumentNullException(
-                        factory, semanticModel.Compilation, parameter))));
+                    throwStatement));
+
+            var close = ifStatement.FindToken(10);
```

Comment:
> 10

😕

Reply:
This whole approach was just to see which elastic markers need to be removed to get the whole thing through the formatter. It also shows that the language agnostic syntax generator approach is just not working is this case.